### PR TITLE
feat: update candy machine preview NFTs to new design

### LINKED ISSUE #104: [Task]: Update candy machine preview NFTs to new design
**Task description.**

image

**Requirements.**
- Empty
- Cache loaded with NFTs grid and cache filename
- Loading status if necessary

### PR DESCRIPTION
**Pull request by VmMad**
Fixes #104

### CODE REVIEW

File: pages/[id].tsx [hunk 1]
```diff
@@ -264,25 +266,46 @@ const CandyMachine: NextPage = () => {
...
+                                        {!nfts.length ? (
+                                            <>
+                                                <h4>NFTs Preview · {itemsRemaining}</h4>
+                                                <Text as='p' className='mt-3 mb-4'>
+                                                    Upload cache file to preview NFTs
+                                                </Text>
+                                            </>
+                                        ) : (
+                                            <>
+                                                <h4>Unminted NFTs - {nfts.length}</h4>
```
Comment (cpl121) [inline pages/[id].tsx:279]: I think that this value can be obtained from the same way as in the nfts header
`${itemsRemaining + mintedNfts.length}`

**Review by cpl121 — CHANGES_REQUESTED** [review]
Nice job!! 
Just one comment

**Review by agus-xyz — APPROVED** [review]
to me, it seems working, but I do not have a cache file to check our cms in mainnet. Could you @aspnxdd please check our cms? It should be easy to try with our cache files or?

**Comment by agus-xyz** [comment]
for some reason, the `Upload Cache file` button is the only one that does not change the mouse on hover. Is this a bug or a feature? 
image
image

**Comment by evavirseda** [comment]
> for some reason, the `Upload Cache file` button is the only one that does not change the mouse on hover. Is this a bug or a feature? image image

That's because it's not a button, `Upload Cache file` is an input, but we can add the hover effect

**Comment by VmMad** [comment]
> for some reason, the `Upload Cache file` button is the only one that does not change the mouse on hover. Is this a bug or a feature? image image

I changed it already, you can check it out now. Other buttons weren't showing pointer either

**Review by agus-xyz — COMMENTED** [review]
I think it would be better having the nft preview reallocated just before the minted nfts. This way the user does not need to wait for the minted nfts to show in order to access the preview. I updated the design accordingly. What do you think?
image

**Comment by agus-xyz** [comment]
@aspnxdd also suggested after testing, that in case the cache file does not have any nft (which might be the case sometimes), it should show `NFTs Preview not available in cache file` or something like that.

**Review by evavirseda — APPROVED** [review]
Nice :rocket: just confilcts!

**Review by cpl121 — APPROVED** [review]

**Comment by evavirseda** [comment]
> @aspnxdd also suggested after testing, that in case the cache file does not have any nft (which might be the case sometimes), it should show `NFTs Preview not available in cache file` or something like that.

@agus-xyz this could be improve when we have notification system

**Pull request merged**